recipes/q/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "q" %}
+{% set version = "3.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Sann5/q/archive/pack_issue_326.zip
+  sha256: 420e92546a4ba42989be609768c67d636097bc2d5054756c7381d7c766a30533
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py>=311]
```

Comment:
I know this was brought up before, but this could be replaced with the following run requirements:

```yaml
run:
  - python >=3.8,<3.11.0dev0
  - __unix
  - six
```

Reply:
Im not sure exactly what you mean. Can you make the change suggestion with the review tool so I can see exactly what you propose? Also then I can quickly accept it if it looks reasonable.

Reply:
Yes. There's no reason to not make this package `noarch`. We let it fail on Windows and then constrain `python` so 3.11 is not supported. Let me add a couple comments.